plasma_framework/contracts/src/vaults/Vault.sol

```diff
@@ -6,17 +6,17 @@ import "../framework/utils/Operated.sol";
 
 /**
  * @notice Base contract for vault implementation
- * @dev This is the functionality to swap "deposit verifier".
- *      By setting new deposit verifier, we can upgrade to a new deposit tx type without upgrading the vault.
+ * @dev This is the functionality to swap "deposit verifier"
+ *      Setting a new deposit verifier allows an upgrade to a new deposit tx type without upgrading the vault
  */
 contract Vault is Operated {
     event SetDepositVerifierCalled(address nextDepositVerifier);
     PlasmaFramework internal framework;
     bytes32[16] internal zeroHashes; // Pre-computes zero hashes to be used for building merkle tree for deposit block
 
     /**
-     * @notice Stores deposit verifier contracts addresses where first was effective upto
-     *  `newDepositVerifierMaturityTimestamp` point of time and second become effective after
+     * @notice Stores deposit verifier contract addresses, where the first contract address is effective until the 
+     *  `newDepositVerifierMaturityTimestamp`, after which the second contract address becomes effective
```

Comment:
I'm guessing this is a moment in time, not an interval, so `after which` doesn't ift.